main.go

```diff
@@ -35,6 +35,7 @@ Zabbix options:
                                 [default: 127.0.0.1].
   -p --port <port>            Port of zabbix server [default: 10051].
   --zabbix-prefix <prefix>    Add part of your prefix for key [default: None].
+	-h --host <host>            Host to send metrics to [default: os.Hostname()]
```

Comment:
Name of the library and its function in help message - looks strange. It would be better if the hostname added automatically as a default value. Check out this commits https://github.com/zarplata/zabbix-agent-extension-rabbitmq/commit/a359106a5914b9623e707d6d85205299db133dcf https://github.com/zarplata/zabbix-agent-extension-rabbitmq/commit/82dc7345a1355e8d89fe93921c0e1511ae57bbba they contain a complete solution to pass actual hostname from Zabbix and I think this is what you want.

Reply:
Thanks. Fixed